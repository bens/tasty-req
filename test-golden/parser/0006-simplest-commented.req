# first line comment
1: GET /  # this is a header
# comment before separator
---  # separator
# after separator
{ # comment in an object
}
# eof
