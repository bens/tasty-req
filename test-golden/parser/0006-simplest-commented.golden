[ Command
    { command'ident = 1
    , command'always = False
    , command'method = "GET"
    , command'url = [ Right "/" ]
    , command'request_body = []
    , command'response_body = [ JsonObject (fromList []) ]
    }
]
